frontend/src/components/layout/navigation/MobileNavigation.module.scss

```diff
@@ -0,0 +1,150 @@
+@import "../../../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.mobile-menu {
+  width: 100%;
+  position: relative;
+  // z-index of 1 (so it overlaps content)
+  // allows menu to slide under main navigation header
+  z-index: 1;
+  flex-direction: column;
+}
+
+#mobile-menu {
```

Comment:
Woah! I was going to say that I didn't think you could use CSS Modules with IDs, but looks like you can - nice find! :exploding_head: